tests/cross/src/test/scala/tests/pc/CompletionScala3Suite.scala

```diff
@@ -71,4 +71,21 @@ class CompletionScala3Suite extends BaseCompletionSuite {
        |""".stripMargin,
     topLines = Some(1),
   )
+
+  checkEdit(
+    "trait-member",
+    """|trait Foo:
+       |  def foo: String
+       |
+       |object Bar extends Foo:
+       |  def@@
```

Comment:
Could you also check if that works in VS Code, it seems we might be setting a wrong filter text

Reply:
<img width="763" alt="Screenshot 2023-06-22 at 10 49 16" src="https://github.com/scalameta/metals/assets/26606662/2c6dd04b-364c-4220-a123-38aeb4670ec3">